worldengine/model/world.py

```diff
@@ -852,7 +851,67 @@ def is_chaparral(self, pos):
     #
 
     def n_actual_plates(self):
-        return self.layers['plates'].data.max() + 1
+        return self.plates.data.max() + 1
+
+    #
+    # Accessors
+    #
+
+    @property
+    def elevation(self):
+        return self.layers['elevation']
```

Comment:
currently many setters take 2 values and we could not replicate that for property setters. For the others, if we implemented propery setters accepting a numpy matrix it would like this:

```
world.ocean = data
data = world.ocean.data
```

So things like:
```
world.ocean = world.ocean
```

Would not work and actually mess up things, you should instead use

```
world.ocean = world.ocean.data
```

So I would stay away from property setters and use explicit setters for now